tests/sql.py

```diff
@@ -19,7 +19,7 @@ def test_delete_returns_affected_rows(self):
             {"id": 3, "val": "baz"}
         ]
         for row in rows:
-            self.db.execute("INSERT INTO cs50(val) VALUES(:val);", val=row["val"])
+        self.db.execute("INSERT INTO cs50(val) VALUES(:val);", val=row["val"])
```

Comment:
Afraid this is incorrectly indented now!

Reply:
Oh sorry, I just inccidentaly copied one row :-D